config.md

```diff
@@ -308,15 +308,26 @@ The semantics are the same as `Path`, `Args` and `Env` in [golang Cmd](https://g
 
 This OPTIONAL property contains arbitrary metadata for the container.
 This information MAY be structured or unstructured.
-Annotations are key-value maps.
+Annotations MUST be key-value maps where both the key and value MUST be strings.
+While the key and value MUST be present, they MAY be empty strings.
+Keys MUST be unique, and best practice is to namespace the keys.
+Keys SHOULD be named using a reverse domain notation - e.g. `com.example.myK
+ey`.
+Keys starting with the `org.opencontainers` namespace are reserved and MUST NOT be used.
```

Comment:
What if someone registers my-opencontainers.org?  I expect we want to claim `org.opencontainers` (which we'd never use, but just to be clear), and anything that starts with `org.opencontainers.`.